scripts/validate/gomod.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+exit_code=0
+
+echo "==> Running go mod verify check <=="
+
+GO111MODULE=on go mod verify || exit_code=1
```

Comment:
I don't think this is doing what we need it to do.

Reply:
I made some changes to the code according to what you initially proposed in #514.

I am facing some error in the builds such as https://circleci.com/gh/hectorj2f/virtual-kubelet/29?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link and https://circleci.com/gh/hectorj2f/virtual-kubelet/28?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link which doesn't seem fully related to any mistake in my changes. Perhaps I am missing something in relation how the CI of this project works.

Besides, I cannot access to the build in https://virtual-kubelet.visualstudio.com/d2adf84f-67a6-4304-a3b5-0235942fdba2/_build/results?buildId=584. So it is hard to debug any potential issue :/.